### PR TITLE
Add inventory size to the slots syntax

### PR DESCRIPTION
**Pull request by TheLimeGlass**
### Description
Add `inventory size of %inventories%` to the slots syntax. Many people don't know inventory size actually exists in Skript. Addons call this term "inventory size", so this pull adds an aliases for it.

---
**Target Minecraft Versions:** any 
**Requirements:** none 
**Related Issues:** none

### CODE REVIEW

**Comment by Fusezion** [comment]
I personally fine the usage of "inventory size" confusing and don't believe this should be something encouraged to use. For me when I see "inventory size" this feels more related to "rows" and not "slots" oh the inventory size of a barrel is 3 makes sense but
reading it as inventory size of a barrel is 27 that's much less understanding as we're not talking about a size of the inventory to me.

And lastly is there any way to possible make the syntax look like less of a mess?
`(holder[s]|viewers|[amount of] rows|([amount of] slots|inventory size)) of %inventories%`

This issue is too prevalent on skript expressions where there's multiple options and should be changed to make reading docs understandable.

**Comment by sovdeeth** [comment]
> And lastly is there any way to possible make the syntax look like less of a mess? `(holder[s]|viewers|[amount of] rows|([amount of] slots|inventory size)) of %inventories%`
> 
> This issue is too prevalent on skript expressions where there's multiple options and should be changed to make reading docs easier and more understandable.

I agree, these should each be different patterns.

File: src/main/java/ch/njol/skript/expressions/ExprInventoryInfo.java [hunk 1]
```diff
@@ -52,8 +52,9 @@ public class ExprInventoryInfo extends SimpleExpression<Object> {
 	
 	static {
 		Skript.registerExpression(ExprInventoryInfo.class, Object.class, ExpressionType.PROPERTY,
-				"(" + HOLDER + "¦holder[s]|" + VIEWERS + "¦viewers|" + ROWS + "¦[amount of] rows|" + SLOTS + "¦[amount of] slots)" + " of %inventories%",
-				"%inventories%'[s] (" + HOLDER + "¦holder[s]|" + VIEWERS + "¦viewers|" + ROWS + "¦[amount of] rows|" + SLOTS + "¦[amount of] slots)");
+				"(" + HOLDER + "¦holder[s]|" + VIEWERS + "¦viewers|" + ROWS + "¦[amount of] rows|" + SLOTS + "¦([amount of] slots|inventory size))" + " of %inventories%",
```
Comment (Moderocky) [inline src/main/java/ch/njol/skript/expressions/ExprInventoryInfo.java:55]: We don't use the old split pipe `¦` parse mark anymore.

File: src/main/java/ch/njol/skript/expressions/ExprInventoryInfo.java [hunk 2]
```diff
@@ -52,8 +52,9 @@ public class ExprInventoryInfo extends SimpleExpression<Object> {
...
+				"%inventories%'[s] (" + HOLDER + "¦holder[s]|" + VIEWERS + "¦viewers|" + ROWS + "¦[amount of] rows|" + SLOTS + "¦([amount of] slots|inventory size))"
```
Comment (Moderocky) [inline src/main/java/ch/njol/skript/expressions/ExprInventoryInfo.java:56]: I think `inventory size` is quite ambiguous and might return what somebody doesn't expect. Having the rows/slots part explicit was probably better.

**Review by Moderocky — CHANGES_REQUESTED** [review]

**Comment by sovdeeth** [comment]
Closing due to inactivity

**Pull request closed without merging**